ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,48 @@
+## Description
+
+Provide a description of your issue.
+For any suggestion, please address them on the [forum](https://forum.freegamedev.net/viewforum.php?f=16).
+For translations, go to [transifex](https://www.transifex.com/supertuxkart/supertuxkart/).
+
+## Steps to reproduce
+
+## Configuration
+
+STK release version :
+
+STK Source (ppa, distribution package, official bin,...) :
+
+System : 
+
+Graphic card : 
+
+CPU : 
+
+Gamepads/keyboards models if related to the issue :
+
+## Additional informations
+
+Please provide stdout.log, it is located in
+
+* %appdata%\supertuxkart\0.8.2 (windows)
+* ~/.config/supertuxkart/0.8.2 (linux)
```

Comment:
*located in:

```
* `%appdata%\supertuxkart\0.8.2` (Windows)
* `~/.config/supertuxkart/0.8.2` (Linux)
```

Also maybe mention Macs.

Reply:
Why do you want to change the formatting ? The backticks are intended for the content of the file.